src/loki/worker.cc

```diff
@@ -71,6 +71,16 @@ void loki_worker_t::parse_costing(Api& api, bool allow_none) {
     throw valhalla_exception_t{124};
   }
 
+  auto costings = options.costings();
+  for (const auto& pair : costings) {
+    if (!allow_hard_exclusions &&
+        (pair.second.options().exclude_bridges() || pair.second.options().exclude_tolls() ||
+         pair.second.options().exclude_tunnels() || pair.second.options().exclude_highways() ||
+         pair.second.options().exclude_ferries())) {
```

Comment:
I wonder if using the singular nouns here would be more in line with the matching `use_*` parameters, although the plural sounds a little better

Reply:
i vote for plurals

Reply:
this is just to keep it seperate from exclude options in the search filter!